components/button.json

```diff
@@ -21,6 +21,25 @@
         "text": "@theme-text-primary-disabled"
       }
     },
+    "primaryDark": {
```

Comment:
what is this? it is exactly the same as the primary button...

Reply:
Pushed commit [Updated common style to button component](https://github.com/momentum-design/tokens/pull/212/commits/bc97b533a6c5a4eb22304b31ac027763ce1c3aad)
Changing this conversation to Resolved.